services/api/src/database.js

```diff
@@ -5,11 +5,19 @@ mongoose.Promise = Promise;
 
 module.exports = async () => {
   await mongoose.connect(config.get('MONGO_URI'), {
-    useFindAndModify: false,
+    // useNewUrlParser: The underlying MongoDB driver has deprecated their current connection string parser.
```

Comment:
nit but I don't think we need the param in the comment... it's pretty standard that the comment is above the relevant line